docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -35,26 +35,26 @@ The endpoint takes a job or list of job ids and then attempts to resubmit them t
 * EE2 looks up the job versions and parameters, and then submits the job to be retried, incrementing the `retry_count`
```

Comment:
job versions? Should that be app version?

Reply:
updated